src/Tizen.NUI/src/internal/Common/PropertyHelper.cs

```diff
@@ -294,18 +294,22 @@ internal KeyFrames RefineKeyFrames(KeyFrames keyFrames)
                 var refined = keyFrames;
                 if (propertyValueConverter != null)
                 {
-                    // TODO Enable this code when csharp-binder is ready
-                    // refined = new KeyFrames();
-                    // for (uint i = 0; i < keyFrames.Count; i++)
-                    // {
-                    //     var keyFrame = keyFrames.GetKeyFrame(i);
-                    //     var newKeyFrame = propertyValueConverter(keyFrame);
-                    //     if (newKeyFrame == null)
-                    //     {
-                    //         return null;
-                    //     }
-                    //     refined.Add(newKeyFrame);
-                    // }
+                    uint keyFramesCount = keyFrames.GetKeyFrameCount();
+                    float keyFrameProgress;
+                    using PropertyValue keyFrameValue = new PropertyValue();
+
+                    refined = new KeyFrames();
+
+                    for (uint i = 0; i < keyFramesCount; i++)
+                    {
+                        keyFrames.GetKeyFrame(i, out keyFrameProgress, keyFrameValue);
+                        var newKeyFrameValue = propertyValueConverter(keyFrameValue);
```

Comment:
newKeyFrameValue도 바로 삭제 될 수 있게 using을 붙여주는 것이 좋을 것 같습니다.
```suggestion
                        using var newKeyFrameValue = propertyValueConverter(keyFrameValue);
```

Reply:
좋은 포인트입니다. 수정했습니다.